src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -162,6 +162,13 @@ Plugin::Plugin() : m_defaultContexts({}) {
     for (auto& config : _impl->m_configs) {
         CustomLayer::LoadFromFile(config_path, config.second.customLayers, true);
     }
+
+    isModelCachingEnabled = false;
```

Comment:
nit) The default value of this is already false, should we set this again?

Reply:
You're right. It's redundant code and removed. Thank you.